antsibull/docs_parsing/ansible_doc.py

```diff
@@ -203,6 +205,24 @@ def get_collection_metadata(venv: t.Union['VenvRunner', 'FakeVenvRunner'],
     return collection_metadata
 
 
+def get_ansible_core_version(venv: t.Union['VenvRunner', 'FakeVenvRunner'],
+                             env: t.Dict[str, str],
+                             ) -> PypiVer:
+    venv_ansible = venv.get_command('ansible')
+    ansible_version_cmd = venv_ansible('--version', _env=env)
+    output = ansible_version_cmd.stdout.decode('utf-8', errors='surrogateescape')
+    first_line = output.splitlines()[0]
+    # `ansible 2.10.0`
+    m = re.match(r'^ansible ([^[ ]+)', first_line)
+    if m:
+        return PypiVer(m.group(1))
+    # `ansible [core 2.12.0.dev0]  (devel cd8e3a2208) last updated 2021/05/19 08:35:55 (GMT +200)`
+    m = re.match(r'^ansible \[[^]]+ ([^]]+)\]', first_line)
+    if m:
+        return PypiVer(m.group(1))
+    raise ParsingError(f'Cannot interpret first line of ansible --version output: "{first_line}"')
```

Comment:
Would this be equivalent?  If so, it's simpler:

```suggestion
def get_ansible_core_version(venv: t.Union['VenvRunner', 'FakeVenvRunner'],
                             env: t.Dict[str, str],
                             ) -> PypiVer:
    venv_python = venv.get_command('python')
    ansible_version_cmd = venv_python('-c', 'import ansible.release; print(ansible.release.__version__)', _env=env)
    output = ansible_version_cmd.stdout.decode('utf-8', errors='surrogateescape').strip()
    return PypiVer(output)
```

Reply:
That should be equivalent, assuming that `ansible.release.__version__` doesn't surprisingly change at some point :)